geocity/apps/submissions/forms.py

```diff
@@ -675,13 +676,14 @@ def get_list_single_field_kwargs(self, field, default_kwargs):
         return {
             **default_kwargs,
             "choices": choices,
+            "widget": Select2Widget(),
         }
 
     def get_list_multiple_field_kwargs(self, field, default_kwargs):
         return {
             **default_kwargs,
             "choices": [(value, value) for value in field.choices.splitlines()],
-            "widget": forms.CheckboxSelectMultiple(),
+            "widget": Select2MultipleWidget(),
```

Comment:
Suggestion : 

Check length of field.choices.splitlines() to know if we should use 
`"widget": Select2MultipleWidget(),`
or
`"widget": forms.CheckboxSelectMultiple(),`